src/Factory.php

```diff
@@ -135,11 +135,12 @@ function ($error) use ($client) {
      * Create Redis client connected to address of given redis instance
      *
      * @param string $target
+     * @param string $master_name allow to connect to a redis sentinel
      * @return Client
      */
-    public function createLazyClient($target)
+    public function createLazyClient($target, $master_name = null)
```

Comment:
This could be moved to the `createClient()` method, perhaps as another query parameter? Do you happen to know how this is handled in other Redis clients?

Reply:
I just lack of time to continue on the other client, currently i'm only using lazy client because it's simpler with DI to not handle connection.
I'm not using an other redis client 😕